packages/perseus-editor/src/diffs/renderer-diff.tsx

```diff
@@ -10,11 +10,14 @@ import _ from "underscore";
 import TextDiff from "./text-diff";
 import WidgetDiff from "./widget-diff";
 
-import type {PerseusRenderer} from "@khanacademy/perseus-core";
+import type {PerseusRenderer, PerseusWidget} from "@khanacademy/perseus-core";
 
 // In diffs, only show the widgetInfo props that can change
-const filterWidgetInfo = function (widgetInfo, showAlignmentOptions: boolean) {
-    const {alignment, graded, options, type} = widgetInfo || {};
```

Comment:
Ouch, this is where we introduced the bug. Previously, if the `widgetInfo` was undefined, we handled it by providing an empty object fallback.